servicecontrol/plugins/saga-audit.md

```diff
@@ -29,33 +29,41 @@ All this information is sent to and stored in ServiceControl. Note that the saga
 This plugin results in an increase in load in several areas
 
  1. Endpoint load in order to capture the required information
- 1. Network load due to the extra information sent to ServiceControl
- 1. ServiceControl load in the areas of ingestion, correlation and data cleanup.
+ 2. Network load due to the extra information sent to ServiceControl
+ 3. ServiceControl load in the areas of ingestion, correlation and data cleanup.
 
 The increase in load is proportional to size of the saga data multiplied by the number of messages the the saga receives. Since both these variables are dependent on the specific saga implementation it is not possible to give accurate predictions on the impact of this load in a production system.
 
 
 ## NuGet packages
 
- * NServiceBus Version 6.x: [ServiceControl.Plugin.NSb6.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb6.SagaAudit)
- * NServiceBus Version 5.x: [ServiceControl.Plugin.Nsb5.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb5.SagaAudit)
- * NServiceBus Version 4.x: [ServiceControl.Plugin.Nsb4.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb4.SagaAudit)
- * NServiceBus Version 3.x: Not Available
+* NServiceBus Version 6.x: [ServiceControl.Plugin.NSb6.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb6.SagaAudit)
+* NServiceBus Version 5.x: [ServiceControl.Plugin.Nsb5.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb5.SagaAudit)
+* NServiceBus Version 4.x: [ServiceControl.Plugin.Nsb4.SagaAudit](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb4.SagaAudit)
+* NServiceBus Version 3.x: Not Available
 
 
 ### Deprecated NuGet
 
 If using the older version of the plugin, namely **ServiceControl.Plugin.SagaAudit** remove the package and replace it with the appropriate plugin based on the NServiceBus version. This package has been deprecated and unlisted.
 
+## Configuration
+
+### ServiceControl Queue
+
+For plugin that targets NServiceBus v6+, it is now possible to configure the ServiceControl queue via code:
```

Comment:
Same as above about detail.  Should this configuration be in a common spot are we just like to it rather than repeating it fro each plugin?